cg/apps/lims/orderform.py

```diff
@@ -16,7 +16,12 @@
     "1604:9",  # Orderform Ready made libraries (RML)
     "1605:8",  # Microbial metagenomes
 ]
-CASE_PROJECT_TYPES = ["mip", "external", "balsamic", "mip_rna"]
+CASE_PROJECT_TYPES = [
+    Pipeline.MIP_DNA.value,
+    "external",
+    Pipeline.BALSAMIC.value,
+    Pipeline.MIP_RNA.value,
+]
```

Comment:
Perhaps you could create a method for Pipeline class to return list of all variables?

Reply:
I'm thinking this might be a place to have another Enum than the Pipeline on, you?

Reply:
Or 
CASE_PROJECT_TYPES = Pipeline.values() + "external" ?